Source/Coinbase/CoinbaseApiAuthenticator.cs

```diff
@@ -57,11 +58,15 @@ public void Authenticate(IRestClient client, IRestRequest request)
             var method = request.Method.ToString().ToUpper(CultureInfo.InvariantCulture);
 
             var body = string.Empty;
-
-            var param = request.Parameters.FirstOrDefault(p => p.Type == ParameterType.RequestBody);
-            if (param != null && param?.Value?.ToString() != "null" && !string.IsNullOrWhiteSpace(param?.Value?.ToString()))
-                body = param.Value.ToString();
-
+			if (request.Method != Method.GET) {
```

Comment:
No you are right it is not needed.  My goal was more to make it clear the case in which we were using PathAndQuery and leave the existing code relatively untouched.